packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -16,6 +16,21 @@
   @include -checkbox-and-radio-colors($input-type: "radio", $args...);
 }
 
+@mixin selected-tile-colors($input-active-color) {
+  background-color: color.adjust(color($input-active-color), $alpha: -0.9);
+  border-color: color($input-active-color);
+
+  @media (forced-colors: active) {
+    border: ButtonText solid units("05");
+  }
+}
+
+@mixin selected-input-colors($input-active-color) {
+  background-color: color($input-active-color);
+  box-shadow: 0 0 0 units($theme-input-select-border-width)
+    color($input-active-color);
+}
+
```

Comment:
## Mixins over extends

I was required to use a mixin over an extend for two reasons

### Cannot extend compound selectors

I attempted to extend the `usa-checkbox__label:checked` class but Sass no longer supports extending compound selectors. This caused a build error with an appropriate error message.

### Passing a parameter

Since `$input-active-color` is defined by parameters passed within the `-checkbox-and-radio-colors` mixin, I was only able to reference it by adding an argument. Mixins do not support this feature.

> Mixins are obviously necessary when you need to configure the styles using [arguments](https://sass-lang.com/documentation/at-rules/mixin/#arguments) [^1]

[^1]: [Sass documentation](https://arc.net/l/quote/jmwwgmqn)